src/libsyntax/feature_gate.rs

```diff
@@ -372,6 +372,9 @@ declare_features! (
 
     // #[doc(cfg(...))]
     (active, doc_cfg, "1.21.0", Some(43781)),
+
+    // Provide clamp functions
+    (active, clamp, "1.21.0", Some(44095)),
```

Comment:
No need for the feature gate in code for lib changes; the attribute is sufficient.